lib/resources/pix.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @name Pix
+ * @description This module exposes functions
+ *              related to the `/pix` path.
+ *
+ * @module balanceOperations
+ **/
+import routes from '../routes'
+import request from '../request'
+
+const create = (opts, body) =>
```

Comment:
```suggestion
/**
 * `POST /pix/keys`
* Creates a pix key from the given payload
 *
 * @param {Object} opts - An options params which
 *                      is usually already bound
 *                      by `connect` functions.
 *
 * @param {Object} body - The payload for the request.
 * @param  {String} body.recipient_id - The recipient_id assoocietated to the  pix key
 *
 * @returns {Promise} Resolves to the result of
 *                    the request or to an error.
 */
const create = (opts, body) =>
```

só faltou adicionar o comentário na função